spacy_transformers/_align.pyx

```diff
@@ -0,0 +1,83 @@
+# cython: infer_types=True, boundscheck=False
+from cython.operator cimport dereference as deref
+from cython.operator cimport preincrement as preinc
```

Comment:
I personally don't think that these aliases help with readability (especially `preinc` reads funny to me) but I don't care strongly if you prefer it like this.

Reply:
This was copied from similar uses in `spacy`, I don't really care either way.

Reply:
I will leave this for now.